vertical-pod-autoscaler/pkg/recommender/input/cluster_feeder.go

```diff
@@ -290,7 +304,8 @@ func (feeder *clusterStateFeeder) GarbageCollectCheckpoints() {
 		}
 		for _, checkpoint := range checkpointList.Items {
 			vpaID := model.VpaID{Namespace: checkpoint.Namespace, VpaName: checkpoint.Spec.VPAObjectName}
-			_, exists := feeder.clusterState.Vpas[vpaID]
+			_, exists := allVPAKeys[vpaID]
```

Comment:
Not sure if that's a simplification worth making, but I felt like if you can avoid `_` parameters then let's do it?

```suggestion
			exists := allVPAKeys[vpaID]
```

Reply:
fixed